libtelemetry/src/main/java/com/mapbox/android/telemetry/MapboxTelemetry.java

```diff
@@ -241,6 +241,10 @@ boolean optLocationOut() {
     return isLocationOpted;
   }
 
+  Boolean isQueueEmpty() {
```

Comment:
i'd suggest to avoid boxed types and use primitives, in this particular case `boolean` is the way to go.